test/erc4626schema.test.js

```diff
@@ -18,15 +18,22 @@ const schema = {
             summary: { type: "string", enum: ["safe", "unsafe"] },
             review: { type: "string", minLength: 1 },
             warnings: { type: "array", items: { type: "string" } },
-            isCompatible: { type: "string", enum: ["true", "false"] },
+            useBufferForSwaps: { type: "string", enum: ["true", "false"] },
+            useUnderlyingForAddRemove: {
+              type: "string",
+              enum: ["true", "false"],
+            },
+            useWrappedForAddRemove: { type: "string", enum: ["true", "false"] },
```

Comment:
The entry here will be based on the template information right?

Reply:
Correct.